### ISSUE TITLE
Plugins not found & config reset to default on new install

### ISSUE THREAD

**Issue by ravedoglv** [hoobs-org/HOOBS#240]
**Description**
Install on new Mac (Mojave). Set up admin account. Installed camera-ffmpeg. Configured plugin. Added homebridge to iphone. Added cameras. All working. Stopped service. Added LaunchDaemon. Opened browser. Went to URL. Wanted me to set up admin account (again) . Used Same credentials. Homebridge dashboard. Homebridge running. No plugins installed. No config except for the default.

Cameras not available on iPhone.

**Version**
beta 70

**Did you upgrade**
Please let us know if you upgraded from a previous version.
- [ ] Yes
- [ X] No

**What device are you using?**
Please let us know the device you are running HOOBS on.
- [ ] HOOBS Box
- [ ] HOOBS SD Card
- [X ] Custom Install

**If this is a custom install, what's your operating system?**
Mac OS Mojave 10.14.6

**List your plugins**
camera-ffmpeg

**Post your config**
{
    "server": {
        "port": 8080,
        "autostart": 0,
        "home_setup_id": "X-HM://0023ISYWYAK7L",
        "polling_seconds": 2
    },
    "client": {
        "default_route": "status",
        "inactive_logoff": 30,
        "theme": "hoobs-light",
        "locale": "en",
        "temp_units": "fahrenheit",
        "country_code": "US",
        "postal_code": "94040"
    },
    "bridge": {
        "name": "HOOBS",
        "port": 51826,
        "pin": "031-45-154",
        "username": "A4:6B:9A:D3:05:D4"
    },
    "description": "",
    "ports": {},
    "accessories": [],
    "platforms": []
}

* Remove any passwords from the config file. This is public.

```
POST CONFIG HERE
```

```
POST LOG HERE
[11/11/2019, 4:49:04 PM] HOOBS listning on port 8080.
[Homebridge] [11/11/2019, 4:49:04 PM] Loaded config.json with 0 accessories and 0 platforms.
[Homebridge] [11/11/2019, 4:49:04 PM] ---
[Homebridge] [11/11/2019, 4:49:04 PM] No plugins found. See the README for information on installing plugins.
[Homebridge] [11/11/2019, 4:49:04 PM] Loading 0 platforms...
[Homebridge] [11/11/2019, 4:49:04 PM] Loading 0 accessories...
[Homebridge] X-HM://0023ISYWYAK7L
[Homebridge] [11/11/2019, 4:49:04 PM] Service is running on port 51826.

**Additional context**
My plugins and the config for them are gone after a restart.

**Comment by ravedoglv** [comment]
Update: After it appeared to be a blank account, I then logged out of the gui. The opened again. This time it allowed me to use my my account and password. When I logged in, I had to added camera plugin again and then configured the plugin. I had to readd the homebridge and all thew cameras to home on iphone again. This time it stuck.

Apparently when you first create an account with hoobs running from the terminal 
hoobs -p 'password for current user'

anything you create is not saved.

When i had run Hoobs as a service with a LaunchDaemon then the setup and config saved. 

Why wouldnt the initial setup when you are running from the terminal not save anything?
